### ISSUE TITLE
Fix the automated tests

### ISSUE THREAD

**Issue by benjifisher** [jpamental/typogrify#12]
When I visit `/admin/config/development/testing` I get the error message

> Fatal error: Class 'Drupal\typogrify\DrupalWebTestCase' not found in /var/www/drupalvm/drupal/web/modules/contrib/typogrify/src/Tests/TypogrifyClassTestCase.php on line 5

This particular error should be easy to fix, but there may be more once that is done.

We should also add more tests, but that can be discussed on a separate issue.
